third_party/docfx-doclet-143274/pom.xml

```diff
@@ -111,6 +112,11 @@
       <artifactId>commons-io</artifactId>
       <version>${apache.commons-io.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-text</artifactId>
+      <version>${apache.commons-text.version}</version>
```

Comment:
I wondered Guava can do the same job but it seems not https://stackoverflow.com/questions/34091304/does-guava-provide-a-method-to-unescape-a-string